authentication/admin.py

```diff
@@ -1,3 +1,8 @@
 from django.contrib import admin
+from authentication.models import User, UserManager
+
+class UserAdmin(admin.ModelAdmin):
+    pass
+
+admin.site.register(User, UserAdmin)
```

Comment:
`@admin.register(User)
class UserAdmin(admin.ModelAdmin):
    pass
`
should look like this